index.html

```diff
@@ -5,10 +5,10 @@
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
 	<meta name="keywords" content="cv, resume, hResume, github, community, free, fun, nice, useful" />
 	<meta name="description" content="GitHub resumes, generated by the community, for the community." />
-	<link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/2.7.0/build/reset-fonts-grids/reset-fonts-grids.css" media="all" /> 
+	<link rel="stylesheet" type="text/css" href="css/reset-fonts-grids.css" media="all" /> 
 	<link rel="stylesheet" type="text/css" href="css/resume.css" media="all" />
 	<link rel="stylesheet" type="text/css" href="css/print.css" media="print" />
-	<script type="text/javascript" src="http://code.jquery.com/jquery-1.8.1.min.js"></script>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.8.1.min.js"></script>
 	<script type="text/javascript" src="js/mustache.js"></script>
```

Comment:
Yes. I agree with you. I have leave a comment at #182, but forgot to remove this comment. Sorry.